hybridCarboniteMetadataService/src/main/scala/cromwell/services/metadata/hybridcarbonite/CarboniteWorkerActor.scala

```diff
@@ -122,17 +122,15 @@ object CarboniteWorkerActor {
   sealed trait CarboniteWorkerMessage
   case class CarboniteWorkflowComplete(workflowId: WorkflowId, result: cromwell.services.metadata.MetadataArchiveStatus) extends CarboniteWorkerMessage
 
-
-  val findWorkflowToCarboniteQueryParameters: Seq[(String, String)] = Seq(
+  def findWorkflowToCarboniteQueryParameters(minimumSummaryEntryId: Option[Long]): Seq[(String, String)] = Seq(
```

Comment:
I'd suggest to rename method to something like `buildParametersForFindWorkflowToCarboniteQuery`

Reply:
Is the purpose of `minimum-summary-entry-id` to indicate to the carboniter this is the summarized workflow to start carboniting, and because `minimum-summary-entry-id` is incremental?

Reply:
It's to avoid turning on carbonite in prod and immediately having a millions-long list of workflows to carbonite. The idea is to have only new workflows get carbonited (at least initially).